File: QuickFly.Server/Models/Reservation.cs
using QuickFly.Server.enums;

namespace QuickFly.Server.Models
{
    public class Reservation
    {
        public Guid Id { get; set; }
        public string Name { get; set; } = default!;

        public string LastName { get; set; } = default!;

        public String FligthNumber { get; set; } = default!;

        public DateTime DepartureDate { get; set; }

        public DateTime LandingDate { get; set; }


        public TicketClass TicketClass { get; set; }


    }
}

File: quickfly.client/src/app/shared/models/reservation.ts
import { DatetimeAdapter } from '@mat-datetimepicker/core';

export interface Reservation {
  id: string;
  name: string;
  lastName: string;
  fligthNumber: string;
  departureDate: Date;
  landingDate: Date;
  ticketClass: number;
}
